arm_diags/src/diurnal_cycle_LAcoupling.py

```diff
@@ -0,0 +1,236 @@
+#===========================================================================================================================
+# Program for generate the diurnal cycle on Land-Atmosphere Coupling
+#---------------------------------------------------------------------------------------------------------------------------
+# V4 Development
+    # ----------------------------------------------------------------------------------------------------
+    # Cheng Tao - Jul2024 @ LLNL
+    # ### Diurnal cycle of SH, LH, T_srf, RH_srf, LCL, PBL
+    # ----------------------------------------------------------------------------------------------------
+#===========================================================================================================================
+import os
+import pdb
+import glob
+import cdms2
+import cdutil
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.gridspec import GridSpec
+from .varid_dict import varid_longname
+from .taylor_diagram import TaylorDiagram
+from .utils import climo
+import MV2
+import matplotlib.gridspec as gridspec
+import scipy.stats
+from scipy import interpolate
+import calendar
+import math
+
+def get_seasonal_3hr(var,seasons,years):
```

Comment:
`get_seasonal_3hr` and `get_seasonal` are repeated in two new component. If they are identical, we can move them under `utils`, also I'm wondering if the existing climo function can provide the same capabilities. Do both functions to get seasonal mean from 3hr and daily datasets?

Reply:
Yes, they are to get the seasonal data from the input time series. We can definitely improve this part if some existing climo functions are available.

Reply:
@chengzhuzhang, I checked climo function. It differs from get_seasonal_3hr and get_seasonal, which outputs the diurnal cycle for each season. I rewrote the function, which merged get_seasonal_3hr and get_seasonal into one function, and moved them under utils. Please check get_diurnal_cycle_seasons under utils.py for details. Thank you!